examples/testdenoise.rs

```diff
@@ -16,15 +16,12 @@ fn main() {
     let mut buffer: [u8; NN * 2] = [0; NN * 2];
 
     let mut st = SpeexPreprocess::new(NN, 8000).unwrap();
-    st.preprocess_ctl(SPEEX_PREPROCESS_SET_DENOISE, 1).unwrap();
-    st.preprocess_ctl(SPEEX_PREPROCESS_SET_AGC, 0).unwrap();
-    st.preprocess_ctl(SPEEX_PREPROCESS_SET_AGC_LEVEL, 8000)
-        .unwrap();
-    st.preprocess_ctl(SPEEX_PREPROCESS_SET_DEREVERB, 0).unwrap();
-    st.preprocess_ctl(SPEEX_PREPROCESS_SET_DEREVERB_DECAY, 0f32)
-        .unwrap();
-    st.preprocess_ctl(SPEEX_PREPROCESS_SET_DEREVERB_LEVEL, 0f32)
-        .unwrap();
+    st.preprocess_ctl(SPEEX_PREPROCESS_SET_DENOISE, &1).unwrap();
```

Comment:
You should not need to pass by reference anymore

Reply:
Sorry, forgot to update. Done.